pkg/migrations/mls/20240109001927_add-messages.up.sql

```diff
@@ -0,0 +1,37 @@
+SET statement_timeout = 0;
+
+--bun:split
+
+CREATE TABLE group_messages (
+    id BIGSERIAL PRIMARY KEY,
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
```

Comment:
Good call with the regular timestamp. Totally fine to not have nanosecond precision here. We aren't even using this for sorting anymore, the timestamps are purely informational.